examples/src/main/python/ml/multiclass_logistic_regression_with_elastic_net.py

```diff
@@ -43,6 +43,43 @@
     # Print the coefficients and intercept for multinomial logistic regression
     print("Coefficients: \n" + str(lrModel.coefficientMatrix))
     print("Intercept: " + str(lrModel.interceptVector))
+
+    trainingSummary = lrModel.summary
+
+    # Obtain the objective per iteration
+    objectiveHistory = trainingSummary.objectiveHistory
+    print("objectiveHistory:")
+    for objective in objectiveHistory:
+        print(objective)
+
+    print("False positive rate by label:")
```

Comment:
Do we want to have a consistent comment as per the Java version above?: `// for multiclass, we can inspect metrics on a per-label basis`

Reply:
Done